aptos-move/aptos-aggregator/src/delayed_field_extension.rs

```diff
@@ -68,7 +79,9 @@ impl DelayedFieldData {
                     DelayedChange::Create(DelayedFieldValue::Aggregator(value)) => {
                         match math.unsigned_add_delta(*value, &input) {
                             Ok(new_value) => {
-                                *value = new_value;
+                                if apply_delta {
```

Comment:
nit: `Ok(new_value) if apply_delta => {}, _ => {}`?

Reply:
return is different?